examples/ecs-fargate/prisma/infra/update.js

```diff
@@ -28,11 +28,14 @@ async function main() {
   console.log('nPushing image...')
   execSync(`docker push ${repo_uri}:latest`);
 
+  const serviceArn = (await prisma.service.findFirst({
+    where: { name: `${APP_NAME}-service`, region: `${REGION}` },
+    select: { arn: true },
+  })).arn.toString();
+
   console.log('Force new deployment')
-  await prisma.ecs_simplified.update({
-    where: { name: APP_NAME },
-    data: { force_new_deployment: true }
-  });
+  await prisma.$queryRaw`SELECT * from deploy_service('${arn}');`;
+
   const commit = await prisma.$queryRaw`SELECT * from iasql_commit();`
```

Comment:
is this commit necessary?

Reply:
I also think this might not be necessary.

Reply:
it was there on the original code. Maybe we remove in a followup?

Reply:
I think it is part of this change, if I'm not wrong it was applying the previous `force_new_deployment=true` change. That is not necessary anymore

Reply:
ah you're right, as the rpc does not need the commit